crates/turborepo-ui/src/logs.rs

```diff
@@ -58,9 +58,12 @@ impl<W: Write> Write for LogWriter<W> {
             (Some(log_file), None) => log_file.write(buf),
             (None, Some(prefixed_writer)) => prefixed_writer.write(buf),
             (None, None) => {
-                // Should this be an error or even a panic?
-                debug!("no log file or prefixed writer");
-                Ok(0)
+                debug!(
+                    "No log file or prefixed writer to write to. This should only happen when \
+                     both caching is disabled and output logs are set to none."
+                );
+
+                Ok(1)
```

Comment:
When I was giving usize of `0` here, I was getting this error:
`× internal errors encountered: failed to write whole buffer`

I'm imagining there's a Rust-y reason for this, but I don't know it and couldn't come up with an answer upon searching.

If you're reading this and know why, please educate me!

Reply:
It's the caller thinking that `0` means a failure at creating/writing to the buffer.